test/mongo_pruner_test.py

```diff
@@ -29,4 +29,4 @@ def test_prune_something(self):
         self.mongo_pruner._stop_event = Mock(wait=Mock(side_effect=[False, True]))
 
         self.mongo_pruner.run()
-        self.assertTrue(self.collection_mock.objects.return_value.delete.called)
+        self.assertTrue(self.collection_mock.objects.return_value.no_cache.called)
```

Comment:
We need to make sure that delete is actually called here... adding an additional check for no_cache being called is good though.

Reply:
Fixed it